content/pt/tutorials/4.moving-from-nuxtjs-dotenv-to-runtime-config.md

```diff
@@ -1,62 +1,62 @@
 ---
 template: post
-title: 'Moving from @nuxtjs/dotenv to runtime config'
-description: 'In our frontend applications, we often use APIs and third-party integrations which require us to use configuration data which is usually provided by environment variables. These variables should not be exposed to the frontend as the browser environment is accessible by all visitors.'
+title: 'Movendo de @nuxtjs/dotenv para configuração de tempo de execução'
+description: 'Em nossas aplicações de frontend, utilizamos APIs com frequência e integrações de terceiros que requerem que nós utilizemos dados de configuração que normalmente são fornecidas pelas variáveis de ambiente. Estas variáveis não devem ser expostas ao frontend visto que o ambiente do navegador é acessível por todos os visitantes.'
 imgUrl: blog/moving-from-nuxtjs-dotenv-to-runtime-config/main.jpeg
 imgCredits: Norris Niman
 imgCreditsUrl: https://unsplash.com/@norrisniman
 date: 2020-06-15
 authors:
   - name: "Debbie O'Brien"
-    avatarUrl: https://pbs.twimg.com/profile_images/1537114604278530048/FMrBa87o_400x400.jpg
+    avatarUrl: https://pbs.twimg.com/profile_images/1252900852156772352/JLIVJ-TC_400x400.jpg
```

Comment:
Why did profile pic change?

Reply:
I already sent an update that will correct this,
sorry to take too much time to fix it.

@ChristopheCVB